__tests__/app.test.js

```diff
@@ -134,4 +134,36 @@ describe("GET /api/articles", () => {
       expect(msg).toBe('bad request')
     })
   })
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test('200: returns an array of all comments related to a particular article, sorted by date', () => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then(({body}) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSortedBy('created_at')
+        comments.forEach((obj) => {
+            expect(obj).toMatchObject({
+                author: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_id: expect.any(Number)
+        })
+      })
+    })
+  })
+  test('400: returns a 400 error if there is a bad request in path', () => {
+    return request(app)
+    .get("/api/articles/banana/comments")
+    .expect(400)
+    .then(({body}) => {
+      const {msg} = body;
+      expect(msg).toBe('bad request')
+    })
+  })
```

Comment:
There are some missing tests here. A 404 for comments on an article that doesn't exist, what status code would it be for an article that does exist but has no comments?